notes/running-mobius-app.md

```diff
@@ -180,6 +180,18 @@ Displays the number of XML elements in the input XML file provided as the first
 `sparkclr-submit.cmd --jars <jar files used for using Hive in Spark> --exe HiveDataFrame.exe C:\Git\Mobius\examples\Sql\HiveDataFrame\bin\Debug`
 
 Reads data from a csv file, creates a Hive table and reads data from it
+
+### Cassandra Example (Sql)
+* Download following jars that are needed to use Spark with Cassandra. **Note** that you need to get the right version of the jar files depending on the versions of Spark and Cassandra. 
```

Comment:
It will be nice to add a URL pointer to Cassandra connnect tutorial/wiki page as a reference, so the reader can go to Cassandra Connector page to read about how to include Cassandra jars or download Cassandra spark packages (the reader may find there is a spark-package option for leveraging Cassandra connector).